internal/annotators/tls_test.go

```diff
@@ -3,14 +3,15 @@ package annotators
 import (
 	"context"
 	"encoding/json"
-	"github.com/project-alvarium/alvarium-sdk-go/pkg/config"
-	"github.com/project-alvarium/alvarium-sdk-go/pkg/contracts"
-	"github.com/project-alvarium/alvarium-sdk-go/test"
 	"io"
 	"io/ioutil"
 	"net/http"
 	"net/http/httptest"
 	"testing"
+
+	"github.com/project-alvarium/alvarium-sdk-go/pkg/config"
+	"github.com/project-alvarium/alvarium-sdk-go/pkg/contracts"
+	"github.com/project-alvarium/alvarium-sdk-go/test"
```

Comment:
Why is this stuff getting moved around?

Reply:
It's the IDE, I'll be fixing all of this before submitting the real PR

Reply:
This is VSCode's Golang extension, it sorts the imports alphabetically also but it lists the built-in packages first then lists the others.